flink-table/flink-table-planner-blink/src/test/resources/org/apache/flink/table/planner/plan/batch/sql/agg/DistinctAggregateTest.xml

```diff
@@ -193,14 +193,14 @@ LogicalAggregate(group=[{}], EXPR$0=[COUNT(DISTINCT $0)], EXPR$1=[SUM(DISTINCT $
     </Resource>
     <Resource name="planAfter">
       <![CDATA[
-SortAggregate(isMerge=[true], select=[Final_COUNT(count$0) AS EXPR$0, Final_SUM(sum$1) AS EXPR$1, Final_MIN(min$2) AS EXPR$2])
+SortAggregate(isMerge=[true], select=[Final_COUNT(count$0) AS EXPR$0, Final_SUM(sum$1) FILTER min$2 AS EXPR$1, Final_MIN(min$2) AS EXPR$2])
```

Comment:
The filter arg for final aggregation was always there, the reason we didn't print that out is because their filter index is not among the input fields, because of the CALCITE-3839, the agg input fields has changed, so for this case, the filter arg just prints out.

I have fired a fix to remove the filter args for global agg.